lib/types.d.ts

```diff
@@ -0,0 +1,23 @@
+declare namespace Renovate {
```

Comment:
Is this file necessary for this PR?

Reply:
The typings: yes
The namespace: can be removed, will possilby collide with global types if removed. Some kind of best practise to separate own typings

Reply:
ok